vue-components/stories/BouncingDots.stories.ts

```diff
@@ -0,0 +1,39 @@
+import BouncingDots from '@/components/BouncingDots';
+import { bouncingDotsSizes } from '@/components/bouncingDots';
+import { Component } from 'vue';
+
+export default {
+    component: BouncingDots,
+    title: '/Loader/BouncingDots',
+};
+
+export function basic( args: { content: string; underlined: boolean } ): Component {
+    return {
+        data(): object {
+            return { args, bouncingDotsSizes };
+        },
+        components: { BouncingDots },
+        props: Object.keys( args ),
+        template: `
+			<div>
+				<div v-for="size in bouncingDotsSizes">
+                    {{ size }}<br><br>
+                    <BouncingDots :size="size"></BouncingDots><br><br>
+				</div>
+            </div>
+		`,
+    };
+}
+
+basic.args = {
+    size: 'small',
+};
+
+basic.argTypes = {
+    position: {
+		control: {
```

Comment:
we can remove the controls, since both sizes are visible on the page

Reply:
This configuration can then be replaced with something like there:

https://github.com/wmde/wikit/blob/65074e872c86613c4326a5332aa8a5fcbee76e55/vue-components/stories/QuantityInput.stories.ts#L199-L201

See also https://storybook.js.org/docs/react/essentials/controls#hide-nocontrols-warning
(and the entire page might be of interest in general)

Reply:
@SaiSan-WMDE thank you for the update
@micgro42 thanks for the resource